3 August, 2017
48 Months - September 2016 - Aug 2020

In [5]:
principal_amount = {1: (9939.89, 12265.60, 12173.33, 12474.96, 12201.21, 12686.35, 13161.43, 12728.34, 12841.01, 13126.82, 12902.49, 13514.52),
                  2: (13304.73, 13422.51, 13694.55, 13811.86, 13639.46, 14047.03, 14442.28, 14024.42, 14411.23, 14534.69, 14538.14, 14783.74), 
                  3: (14797.70, 15037.16, 15061.81, 15295.01, 15235.03, 15556.55, 15863.23, 15743.53, 15882.90, 16096.64, 16165.99, 16373.03),
                  4: (16394.77, 16708.31, 16747.07, 16895.31, 17044.88, 17231.09, 17378.71, 17527.58, 17657.29, 17829.00, 17971.42, 18135.40)
}


In [6]:
# per year mark-up
mark_up = {1: (8351.16, 6025.45, 6117.72, 5816.09, 6089.84, 5604.70, 5129.62, 5562.71, 5450.04, 5164.23, 5388.56, 4776.53),
          2: (4986.32, 4868.54, 4596.50, 4479.19, 4651.59, 4244.02, 3848.77, 4266.63, 3879.82, 3756.36, 3752.91, 3507.31), 
          3: (3493.35, 3253.89, 3229.24, 2996.04, 3056.02, 2734.50, 2472.82, 2547.52, 2408.15, 2194.41, 2125.06, 1918.02),
          4: (1896.28, 1582.74, 1543.98, 1395.74, 1246.17, 1059.96, 912.34, 763.47, 633.76, 462.05, 319.63, 155.36)
}
    

In [7]:
sum(mark_up[1])


69476.65

In [8]:
sum(mark_up[1]+mark_up[2])

120314.61000000002

In [10]:
mark_up.values()

[(8351.16,
  6025.45,
  6117.72,
  5816.09,
  6089.84,
  5604.7,
  5129.62,
  5562.71,
  5450.04,
  5164.23,
  5388.56,
  4776.53),
 (4986.32,
  4868.54,
  4596.5,
  4479.19,
  4651.59,
  4244.02,
  3848.77,
  4266.63,
  3879.82,
  3756.36,
  3752.91,
  3507.31),
 (3493.35,
  3253.89,
  3229.24,
  2996.04,
  3056.02,
  2734.5,
  2472.82,
  2547.52,
  2408.15,
  2194.41,
  2125.06,
  1918.02),
 (1896.28,
  1582.74,
  1543.98,
  1395.74,
  1246.17,
  1059.96,
  912.34,
  763.47,
  633.76,
  462.05,
  319.63,
  155.36)]

In [19]:
per_year_markup = {k:sum(v) for k, v in mark_up.iteritems() }
per_year_markup
    

{1: 69476.65,
 2: 50837.95999999999,
 3: 32429.020000000004,
 4: 11971.479999999998}

In [20]:
per_year_pp = {k:sum(v) for k, v in principal_amount.iteritems() }
per_year_pp

{1: 150015.94999999995,
 2: 168654.63999999996,
 3: 187108.58,
 4: 207520.83000000005}

In [22]:
total_pp = sum(per_year_pp.values())
total_pp

713300.0

In [24]:
total_markup = sum(per_year_markup.values())
total_markup

164715.11000000002

# As on 3 August, 2017 - 12 installments paid

In [27]:
markup_to_pay = sum(per_year_markup.values()[1:])
markup_to_pay

95238.45999999999

In [28]:
pp_to_pay = sum(per_year_pp.values()[1:])
pp_to_pay

563284.05

In [31]:
markup_paid = per_year_markup.values()[:1]
markup_paid

[69476.65]

In [46]:
pp_paid = per_year_pp.values()[:1]
pp_paid

[150015.94999999995]

Principal paid = 150,015.95

Mark up paid = 69,476.65

Principal amount pending = 563,284.05

# Prepayment - for 2nd year, 5% of the remaining principal amount

In [39]:
prepayment_charges = (5 * pp_to_pay)/100
prepayment_charges

28164.2025

In [40]:
current_resale = 1150000  # current market price

In [42]:
can_be_saved = markup_to_pay - prepayment_charges
can_be_saved


67074.25749999999

In [43]:
prepayment_to_pay = prepayment_charges + pp_to_pay
prepayment_to_pay

591448.2525000001

In [45]:
cash_in_hand = current_resale - prepayment_to_pay
cash_in_hand

558551.7474999999

Amount to pay with 5% prepayment charges = 591,448.25

Cash after selling the car = 558,551.75


